samples/boards/sensortile_box_pro/sensors-on-board/src/main.c

```diff
@@ -0,0 +1,301 @@
+/*
+ * Copyright (c) 2023 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/sys/printk.h>
+
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/led.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/spi.h>
+#include <zephyr/drivers/sensor.h>
+
+#include <stdio.h>
+
+#define WHOAMI_REG      0x0F
```

Comment:
also unused?

Reply:
looks so... Sorry, I didn't catch it before.